docker/pypi/wmagent-couchdb/couchdb-docker-run.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+### This script is to be used for running the Couchdb docker container at a VM
+### Its sole purpose is to set all the needed mount points from the Host VM and
+### forward all Couchdb runtime parameters to the Couchdb container entrypoint run.sh
+### It accepts only the set of parameters relevant to Couchdb's container run.sh
+### and no build dependent ones. The docker image tag to be searched for execution is
+### always `latest`.
+
+
+# NOTE: In the help call to the current scrit we only repeat the help and usage
+#       information for all the parameters accepted by run.sh.
+help(){
+    echo -e $*
+    cat <<EOF
+
+The script to be used for running a Couchdb docker container at a VM. The full set of arguments
+passed to the current script are to be forwarded to the Couchdb container entrypoint 'run.sh'
+
+Usage: couchdb-docker-run.sh [-t <team_name>] [-n <agent_number>] [-f <db_flavour>]
+
+    -p <pull_image>   Pull the image from registry.cern.ch
+    -t <couchdb_tag>  The Couchdb version/tag to be downloaded from registry.cern.ch [Default:latest]
+    -h <help>
+
+Example: ./couchdb-docker-run.sh -t 3.2.2
+
+EOF
+}
+
+usage(){
+    help $*
+    exit 1
+}
+
+PULL=false
+COUCH_TAG=3.2.2
+
+
+### Argument parsing:
+while getopts ":t:hp" opt; do
+    case ${opt} in
+        t) COUCH_TAG=$OPTARG ;;
+        p) PULL=true ;;
+        h) help; exit $? ;;
+        : )
+            msg="Invalid Option: -$OPTARG requires an argument"
+            usage "$msg" ;;
+    esac
+done
+
+
+couchdbUser=cmst1
```

Comment:
This is not covering non-cmst1 users deploying this service yet, right?

Reply:
yes that's  correct .. We have to wait to see what the other's teams preferences are before we  set this to a user picked up at runtime.